politeiawww/api/cms/v1/v1.go

```diff
@@ -132,9 +134,13 @@ const (
 	// values for each line item in the CSV.
 	PolicyInvoiceFieldDelimiterChar rune = ','
 
+	// PolicySupervisorUserIDSeperator is the character that separates
+	// multiple SupervisorUserIDs for a given cms user.
+	PolicySupervisorUserIDSeperator rune = ','
```

Comment:
Definitely possible, but then you would still need to create the string when going in/out of the db.   Since we won't have any slices in the db itself we'll still need a separator at that point.

Reply:
That's fine.  I think it would make the code more clear that way and you wouldn't need an API policy for it.

Reply:
OK will change that.